src/scene/shader-lib/chunks/chunks.js

```diff
@@ -27,6 +27,8 @@ import cookiePS from './lit/frag/cookie.js';
 import cubeMapProjectBoxPS from './lit/frag/cubeMapProjectBox.js';
 import cubeMapProjectNonePS from './lit/frag/cubeMapProjectNone.js';
 import cubeMapRotatePS from './lit/frag/cubeMapRotate.js';
+import debugOutputPS from './lit/frag/debug-output.js';
+import debugProcessFrontendPS from './lit/frag/debug-process-frontend.js';
```

Comment:
Actually, can we consider this naming too please (since once chunk names are chosen, they can be used in app code). I'm not sure about `debugOutput` - the `_DEBUG_` naming has gone from the constants. I can think of non-debug uses for this. And also `debugProcessFrontend` is a bit verbose. Not sure what could be better though. Ideas, @GSterbrant and @slimbuck?

Reply:
debugOutputPS could be: **postEndPS**
debugProcessFrontendPS could be: **postFrontEndPS** or processLitArgsPS

I'd probably vote for 1st on these lists.

Reply:
Yeah, I think they're better. Would still love for @GSterbrant and @slimbuck to chip in too...

Reply:
We chatted about this offline and decided this naming isn't great and we'd like to fix it, but not in the PR.